apps/server/src/schema/feedback/feedback-operation.ts

```diff
@@ -0,0 +1,55 @@
+import { FeedbackType, prisma } from '@repo/database';
+import { FireAndForget } from '@repo/utils';
+import { GraphQLError } from 'graphql';
+import { builder } from '../builder';
+import { sendFeedbackReceivedEmail } from '../../email';
+import { FeedbackDto, FeedbackTypeEnum } from './feedback-types';
+
+const fireAndForget = new FireAndForget();
+
+builder.mutationFields((t) => ({
+  sendFeedback: t.withAuth({ authenticated: true }).field({
```

Comment:
```suggestion
  sendFeedback: t.withAuth({ authenticated: true }).prismaField({
```
If you return the result of a prisma operation, let's return a prisma field